var/spack/repos/builtin/packages/libefence/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libefence(MakefilePackage):
+    """Electric Fence (or eFence) is a memory debugger written by Bruce Perens.
+    It consists of a library which programmers can link into their code to override
+    the C standard library memory management functions. eFence triggers a program
+    crash when the memory error occurs, so a debugger can be used to inspect the
+    code that caused the error."""
+
+    homepage = "https://en.wikipedia.org/wiki/Electric_Fence"
```

Comment:
🤔 Would https://packages.debian.org/unstable/electric-fence be better?

Reply:
FWIW. There are two other built-in packages using `Wikipedia` as the home page:  `gconf` and `libc`.

Reply:
There is also precedence for using an `unstable` `debian` package directory:  `libpaper`.